src/services/cli-http-client.js

```diff
@@ -64,12 +65,16 @@ class CliRequestClient {
       const b64Auth = Buffer.from(`${opts.username}:${opts.password}`).toString('base64');
       headers.Authorization = `Basic ${b64Auth}`;
     }
-
-    const componentInfo = (headers['User-Agent'] || '').replace(' (', '|').replace(')', '').split('|');
-    componentInfo.push(`${os.platform()} ${os.release()} ${os.arch()}`);
+    const componentInfo = [];
+    componentInfo.push(`(${os.platform()} ${os.arch()})`);
+    const userAgentArr = (headers['User-Agent'] || '').split(' ');
+    componentInfo.push(userAgentArr[0]); // extensions
+    componentInfo.push(userAgentArr[3]); // extensions
```

Comment:
Don't we need to check for array length? Because the case where `headers['User-Agent'] `is not present `userAgentArr` has length 1.

Reply:
We should. The only reason didn't add it was as we're building up the user-agent string [here](https://github.com/twilio/twilio-cli-core/blob/c628f6be20b0a5c3b0b418d0fb43a282c2f2188d/src/services/twilio-api/twilio-client.js#L173), so I couldn't come up with a case where it could be empty.